[IMP] Payroll: Updating main doc (#14187)
odoo/documentation: content/applications/hr/payroll.rst

Payroll
=======

Odoo's **Payroll** app is used to calculate employee's worked time in conjunction with other Odoo
apps, such as **Employees**, **Time Off**, **Attendances**, and **Planning**, and create
corresponding payslips.

When an employee is hired, they must sign a :doc:`contract <payroll/contracts>`. The contract
outlines how the employee is compensated, including the salary structure and rules their pay is
calculated from, when they work (their `working schedule`), and what benefits they receive.

.. important::
   It is critical to install the correct :doc:`country-specific localization
   <payroll/payroll_localizations>`, as this configures all local and national rules, regulations,
   and applicable taxes for the selected country.

In addition to the employee contract, any :doc:`salary attachments <payroll/salary_attachments>`
must be configured before running payroll.

When it is time to run payroll according to the business's schedule (monthly, weekly, bi-weekly,

> etc,), :doc:`payslips <payroll/payslips>` can be created individually or in :doc:`groups

Reviewer: ```suggestion
etc), :doc:`payslips <payroll/payslips>` can be created individually or in :doc:`groups
```

<payroll/batches>`. Odoo calculates when the employees worked based on their :doc:`work entries
<payroll/work_entries>`. Work entries are created according to their :doc:`contracts
<payroll/contracts>`, based on information from the **Attendances**, **Planning**, or **Timesheets**
apps. Any errors with work entries **must** be resolved *before* continuing to process payroll.

After all issues or conflicts are resolved, payslips are :ref:`processed <payroll/process>` then
:ref:`employees are paid <payroll/pay-employee>`, either with a wire transfer or a physical check.

.. note::
   It is possible to pay employees with cash, but this is not typical.

Settings
========

Before running payroll, various settings for the **Payroll** app **must** be configured. Navigate to
:menuselection:`Payroll app --> Configuration --> Settings`. The various settings for
:ref:`accounting <payroll-accounting>`, :ref:`localizations <payroll-localizations>`, :ref:`time off
<payroll-time-off>`, and :ref:`general payroll settings <payroll-payroll-settings>` are specified
here.

.. _payroll-accounting:

Accounting
----------

Configure the following in the accounting section of the configuration menu:

> - :guilabel:`Payroll Entries`: Enable this option to post payroll slips in accounting.

Reviewer: What does "in accounting" mean? In the app? with the accounting department?

- :guilabel:`Payroll SEPA`: Enable this option to create :abbr:`SEPA (Single Euro Payments Area)`
  payments.
- :guilabel:`Batch Account Move Lines`: Enable this option to have a single account move line
  created from all the accounting entries of the same period. This anonymizes accounting entries and
  disables the creation of individual payments.

.. _payroll-localizations:

Localizations
-------------

*Localizations* are country-specific settings preconfigured in Odoo at the creation of the database,
which account for all taxes, fees, and allowances for that particular country.

The :guilabel:`Localization` section of the **Payroll** app :guilabel:`Settings` page includes
specific settings that need to be configured for the specific country.

The settings and options shown in this section vary, depending on the :doc:`localization enabled
<payroll/payroll_localizations>` for the database.

.. warning::

>    It is **not** recommended to alter the localization settings, unless specifically required.

Reviewer: When would you be required? Unless you can think of a reason, I would leave that part off

.. note::
   Odoo can handle multi-company configurations. This is generally done when there is a main company
   or office location, such as a headquarters, with other offices or branches located either in the
   country or internationally, that fall under that main company or headquarters. In Odoo, each
   company, including the headquarters, **must** be set up as their own company or branch using the
   multi-company method.

>    Each individual company can have a different localization setting, since locations can vary

Reviewer: ```suggestion
   Each company can have a different localization setting, since locations can vary
```

   anywhere in the world, where rules and laws differ.

   For more information on companies, refer to the :doc:`Companies <../general/companies>`
   documentation, which covers how to set up multiple companies.

.. seealso::
   :doc:`payroll/payroll_localizations`

.. _payroll-time-off:

Time off
--------

> - :guilabel:`Deferred Time Off`: If time off is taken *after* payslips are validated, the time off

Reviewer: since there is only one bullet point i would remove that

>   must be applied to the following pay period to avoid refunding then reprocessing payslips. Select

Reviewer: ```suggestion
  must be applied to the following pay period to avoid issuing a refund and then reprocessing payslips. Select
```
Would it be considered a refund? The employee would be overpaid at that point

Author: True- they woudl cancel the check/reverse the payment, and reissue it after calculating it (which is why it is avoided at all costs)

     period).

.. _payroll-payroll-settings:

Payroll
-------

The payroll section allows for the installation of a :doc:`payroll localization
<payroll/payroll_localizations>`. Click :icon:`oi-arrow-right` :guilabel:`Choose a Payroll
Localization` and a Kanban view of all available payroll localizations loads. Click
:guilabel:`Install` on the desired localization to install it. If a localization *has* been
installed, only a :guilabel:`Module Info` button appears for that specific localization.

Once a localization has been installed, configure the following fields:

- :guilabel:`Contract Expiration Notice Period`: Enter the number of :guilabel:`Days` before a
  contract expires, when Odoo notifies the responsible person about the upcoming expiration.
- :guilabel:`Payslip PDF Display`: Enable this option to show the payslip's PDF when the state is
  validated.
- :guilabel:`Work Permit Expiration Notice Period`: Enter the number of :guilabel:`Days` before a
  work permit expires, when Odoo notifies the responsible person about the upcoming expiration.
- :guilabel:`YTD Reset Date`: Enter the date when the :abbr:`YTD (Year To Date)` is reset to. By

>   default, this field is set to January first.

Reviewer: ```suggestion
  default, this field is set to January 1st.
```
